airflow/models/__init__.py

```diff
@@ -4290,7 +4291,13 @@ def setdefault(cls, key, default, deserialize_json=False):
 
     @classmethod
     @provide_session
-    def get(cls, key, default_var=__NO_DEFAULT_SENTINEL, deserialize_json=False, session=None):
+    def get(
+        cls,
+        key,  # type: str
+        default_var=__NO_DEFAULT_SENTINEL,  # type: Any
+        deserialize_json=False,  # type: bool
+        session=None
```

Comment:
General style point:

On these "shorter" fns is it possible to put the type annotation at the end instead of inline? Or is this the more commonly accepted style for type comments on func args?

Reply:
Either way works. From https://mypy.readthedocs.io/en/latest/python2.html, we can annotate a function like this:

```python
def hello(): # type: () -> None
    print 'hello'
```

In this case, the arguments and the type annotation wouldn't fit on a single line, so I thought it made sense to split the arguments up, but I'd be happy to reformat.

Reply:
For something as monstorous as the BaseOperator it makes sense to have the comment on the arg, not at the end.

I was just thinking that in this case where there's about "4 or 5" args that it might be clear and shorter to have one type comment at the end. But I'm not sure that is true.

Anyone have any opinions either way?

Reply:
I think this is a little unwieldy for one line:

```python
def get(cls, key, default_var=__NO_DEFAULT_SENTINEL, deserialize=False, session=None):  # type: (str, Any, bool, Any) -> Any
```

We could split the definition over a few lines, but then we're getting close to the original formatting:

```python
def get(
    cls, key, default_var=__NO_DEFAULT_SENTINEL, deserialize=False, session=None
):  # type: (str, Any, bool, Any) -> Any
```

I don't have a strong preference between the second example and the current code. But I think it would be even better to adopt https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-6+Enforce+the+usage+of+a+code+formatter and avoid having these conversations.

Reply:
Yeah, computer enforce style guide ftw.